mesa/experimental/devs/simulator.py

```diff
@@ -228,7 +274,7 @@ def setup(self, model):
 
         """
         super().setup(model)
-        self.schedule_event_now(self.model.step, priority=Priority.HIGH)
+        self.schedule_event_next_tick(self.model.step, priority=Priority.HIGH)
```

Comment:
This is the t=0 to t=1 starting time right?

Reply:
yes